artifacthub/library/general/uniqueserviceselector/%!s(<nil>)/artifacthub-pkg.yml

```diff
@@ -0,0 +1,25 @@
+version: '%!s(<nil>)'
```

Comment:
looks like an issue with version on this one

Reply:
fixed